integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Installer.java

```diff
@@ -105,6 +105,7 @@ public static InstallParams defaultInstallWitParams() {
    * @return an InstallParams instance
    */
   public static InstallParams defaultInstallWleParams() {
+    System.out.println("=============================WLE_DOWNLOAD_URL: " + WLE_DOWNLOAD_URL);
```

Comment:
Can you use logger.info("Resolved WLE_DOWNLOAD_URL is {0}", WLE_DOWNLOAD_URL);

Reply:
this is a debugger. I removed it.